File: parse_pdfs.py
import os
from pypdf import PdfReader

folder = 'public/sertif'
files = [
    'sertif modul 1.pdf',
    'sertif modul 2.pdf',
    'sertif modul 3.pdf',
    'sertif modul 4.pdf'
]

for file in files:
    path = os.path.join(folder, file)
    try:
        reader = PdfReader(path)
        page = reader.pages[0]
        text = page.extract_text()
        print(f"--- {file} ---")
        print(text[:200].replace('\n', ' '))
    except Exception as e:
        print(f"Error reading {file}: {e}")

File: parse_pdfs.js
const fs = require('fs');
const pdf = require('pdf-parse');
const path = require('path');

async function parseAll() {
  const dir = path.join(__dirname, 'public', 'sertif');
  const files = ['sertif modul 1.pdf', 'sertif modul 2.pdf', 'sertif modul 3.pdf', 'sertif modul 4.pdf'];
  
  for (const file of files) {
    const dataBuffer = fs.readFileSync(path.join(dir, file));
    try {
      const data = await pdf(dataBuffer);
      console.log(`--- ${file} ---`);
      console.log(data.text.substring(0, 500).replace(/\n/g, ' '));
    } catch(e) {
      console.log(e);
    }
  }
}

parseAll().catch(console.error);

parseAll().catch(console.error);
